src/Repository/ActivityRepository.php

```diff
@@ -76,43 +116,15 @@ public function findUpcomingByGroup($groups)
     public function findVisibleUpcomingByGroup($groups)
     {
         return $this->createQueryBuilder('p')
+            ->andWhere('p.isArchived = false')
             ->andWhere('p.end > CURRENT_TIMESTAMP()')
             ->andWhere('(p.target IN (:groups)) OR (p.target is NULL)')
+            ->andWhere('p.isArchived != true')
```

Comment:
Duplicate addition? Please fix

Reply:
no idea why I did it twice that way, consider it fixed